### PR TITLE
Added fingerprint values for 2019 Forester Touring. Previous fingerprint was missing a few messages.

### PR DESCRIPTION
**Pull request by john-SRD**

### CODE REVIEW

**Comment by jcak** [comment]
hey @john-SRD, this doesn't compile. you're missing a comma.

Also probably will need to combine with #2763 (updating the same forester fingerprint rather than making a new one)

**Comment by john-SRD** [comment]
Hello @jcak,

I added the comma. Happens way too often with me. Sorry about that.

I'm not sure about combining with #2763. That fingerprint appears a few messages short. I'll leave the judgement for that up to you.

Thank you.

**Comment by jahanzeeb** [comment]
Ok sand link

File: selfdrive/car/subaru/values.py [hunk 1]
```diff
@@ -33,6 +33,10 @@ class CAR:
   # Forester 2019
   {
     2: 8, 64: 8, 65: 8, 72: 8, 73: 8, 280: 8, 281: 8, 282: 8, 290: 8, 312: 8, 313: 8, 314: 8, 315: 8, 316: 8, 326: 8, 372: 8, 544: 8, 545: 8, 546: 8, 554: 8, 557: 8, 576: 8, 577: 8, 722: 8, 801: 8, 802: 8, 803: 8, 805: 8, 808: 8, 811: 8, 826: 8, 837: 8, 838: 8, 839: 8, 842: 8, 912: 8, 915: 8, 940: 8, 1614: 8, 1617: 8, 1632: 8, 1650: 8, 1651: 8, 1657: 8, 1658: 8, 1677: 8, 1722: 8, 1759: 8, 1787: 5, 1788: 8
+  },
+  # Forester Touring 2019
+  {
+    2: 8, 64: 8, 65: 8, 72: 8, 73: 8, 280: 8, 281: 8, 282: 8, 290: 8, 312: 8, 313: 8, 314: 8, 315: 8, 316: 8, 326: 8, 372: 8, 544: 8, 545: 8, 546: 8, 552: 8, 554: 8, 557: 8, 576: 8, 577: 8, 722: 8, 801: 8, 802: 8, 803: 8, 805: 8, 808: 8, 811: 8, 816: 8, 826: 8, 837: 8, 838: 8, 839: 8, 842: 8, 912: 8, 915: 8, 940: 8, 961: 8, 1614: 8, 1617: 8, 1632: 8, 1650: 8, 1651: 8, 1657: 8, 1658: 8, 1677: 8, 1697: 8, 1698: 8, 1722: 8, 1743: 8, 1759: 8, 1787: 5, 1788: 8, 1809: 8, 1813: 8, 1817: 8, 1821: 8, 1840: 8, 1848: 8, 1924: 8, 1932: 8, 1952: 8, 1960: 8
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/values.py:39]: replace the existing fingerprint with this one, it's just a supserset of the existing one

**Comment by jcak** [comment]
should be ok to cancel this pull request. Merged these extra messages you got into the main forester fingerprint as part of #2763

**Comment by adeebshihadeh** [comment]
Thanks for the PR! All the messages were added in d3d57b0744de5ff04c86e91c8a63d92ba681caaa.

**Pull request closed without merging**